### ISSUE TITLE
Cannot reproduce ReviewKD ACC with ResNet50 to MobileNetV2 on CIFAR100

### ISSUE THREAD

**Issue by NickiJQ** [megvii-research/mdistiller#39]
I used this yaml and get 66.82 ACC@1 on CIFAR100（69.89 on paper）, could you give me some suggestion about yaml or other noteworthy details？

EXPERIMENT:
  NAME: ""
  TAG: "reviewkd,res50,mv2"
  PROJECT: "cifar100_baselines"
DISTILLER:
  TYPE: "REVIEWKD"
  TEACHER: "ResNet50"
  STUDENT: "MobileNetV2"
REVIEWKD:
  REVIEWKD_WEIGHT: 7.0
  SHAPES: [1, 2, 4, 8, 16]
  OUT_SHAPES: [1, 4, 8, 16, 32]
  IN_CHANNELS: [12, 16, 48, 160, 1280]
  OUT_CHANNELS: [256, 512, 1024, 2048, 2048]
SOLVER:
  BATCH_SIZE: 128
  EPOCHS: 240
  LR: 0.05
  LR_DECAY_STAGES: [150, 180, 210]
  LR_DECAY_RATE: 0.1
  WEIGHT_DECAY: 0.0005
  MOMENTUM: 0.9
  TYPE: "SGD"

OS: Ubuntu 20.04
GPU: V100 32G

**Comment by Zzzzz1** [comment]
The initial learning rate should be set as 0.01 when training MobileNet and ShuffleNet series.

**Comment by NickiJQ** [comment]
I set LR = 0.01 and get even worse ACC 63.46😂

**Comment by Zzzzz1** [comment]
> I set LR = 0.01 and get even worse ACC 63.46😂

Since you set batch-size as 128, the LR should be 0.02.

**Comment by NickiJQ** [comment]
Finally, I set LR=0.01, BATCH_SIZE=64 and REVIEWKD_WEIGHT=30.0, get ACC 69.88
